docs/Get-WslDistributionOnline.md

```diff
@@ -0,0 +1,90 @@
+---
+external help file: Wsl-help.xml
+Module Name: Wsl
+online version: https://github.com/SvenGroot/WslManagementPS/blob/main/docs/Get-WslDistributionOnline.md
+schema: 2.0.0
+---
+
+# Get-WslDistributionOnline
+
+## SYNOPSIS
+
+Gets information about WSL distributions available from online sources.
+
+## SYNTAX
+
+```
+Get-WslDistributionOnline [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+The `Get-WslDistributionOnline` cmdlet gets information about all the WSL distributions available from online sources.
```

Comment:
Please use a maximum line width of 100 characters in Markdown files.

Reply:
Updated.